experimental/dds/tree2/src/feature-libraries/typed-schema/flexList.ts

```diff
@@ -3,28 +3,61 @@
  * Licensed under the MIT License.
  */
 
+/** Used by `markEager`and `isMarkedEager` */
```

Comment:
Nit (since it doesn't describe the semantics of the item):
```suggestion
/** @remarks Used by `markEager`and `isMarkedEager` */
```

Reply:
Good idea!